Feature/extend textile issue quickinfo

romanolicker opened the pull request.

Commit:
cut of the description with "…" in issue textile quickinfo links

Commit:
replace small "-" with the large "—" one

apfelfabrik commented:
You used the em dash here. According to http://en.wikipedia.org/wiki/Dash#Ranges_of_values, this really should be an en dash, i.e. this character: –

Other than that, code seems good.

apfelfabrik committed:
Uses en dash to seperate start and end date.

According to http://en.wikipedia.org/wiki/Dash#Ranges_of_values, this
really should be an en dash, i.e. this character: –

apfelfabrik commented:
I changed it, you merge it.
